wordpress.org/public_html/wp-content/themes/pub/wporg-support/helphub-update-php-strings.php

```diff
@@ -63,13 +63,20 @@ function wporg_support_filter_update_php_content( $content ) {
 	$content .= '<p>' . __( 'There are two main benefits to keeping PHP up-to-date:', 'wporg-forums' ) . '</p>';
 	$content .= '<ul>';
 	$content .= '<li>';
+	$content .= sprintf(
+		/* translators: 1: link URL to WordPress requirements page, 2: recommended PHP version */
+		__( '<strong>Your website will be faster</strong> because the latest version of PHP is more efficient. Updating to <a href="%1$s">WordPress&#8217;s recommended PHP version</a> (currently %2$s or higher) can deliver a huge performance increase; up to 3 or 4x faster than older versions.', 'wporg-forums' ),
```

Comment:
"Your website may be faster" :) No guarantees, may be slower too because a plugin doesn't work well with the never PHP version and fills thew logs with errors.

Also, "can deliver a huge performance increase; up to 3 or 4x faster than older versions". This is true for PHP itself, not for the website as a whole. PHP is actually pretty fast compared with network speeds, etc. so the performance increase is usually not noticeable. The users will be looking at website performance, not timing PHP :)

So thinking here it would be better to be a bit more modest. Perhaps something like "can deliver a performance increase that will benefit all visitors to your website."

Reply:
Thanks, @azaozz! These claims have been dampened down a bit in 441652d.